src/Symfony/Component/HttpClient/HttpClient.php

```diff
@@ -49,7 +49,7 @@ public static function create(array $defaultOptions = [], int $maxHostConnection
             }
         }
 
-        if (\extension_loaded('curl')) {
+        if (\extension_loaded('curl') && \function_exists('curl_multi_exec')) {
```

Comment:
when does this happen: having curl but not the function?

Reply:
We have the case on the servers delivered by our host. They disable it for security reasons.
This often also happens with shared servers

Reply:
I don't get how disabling this function can improve security. Can't you ask them to re-enable this instead?